test/fixtures/includetwice.jl

```diff
@@ -0,0 +1,6 @@
+module A
+    include("include2.jl")
+end
+module B
+    include("include2.jl")
+end
```

Comment:
We may also want to have a linter for cases like:
```julia
module A
    include("include2.jl")
    # [...]
    include("include2.jl")
end
```
maybe using `included_files` information combined with currently-analyzed module context, but that should be different from `RecursiveIncludeErrorReport`.